catalyst-gateway/bin/src/db/index/schema/cql/txo_assets_by_stake_table.cql

```diff
@@ -6,12 +6,12 @@ CREATE TABLE IF NOT EXISTS txo_assets_by_stake (
     slot_no         varint,   -- slot number the txo was created in.
     txn             smallint, -- Which Transaction in the Slot is the TXO.
     txo             smallint, -- offset in the txo list of the transaction the txo is in.
-    policy_id       blob,     -- asset policy hash (id) (28 byte binary hash)
-    policy_name     text,     -- name of the policy (UTF8) TODO: https://github.com/input-output-hk/catalyst-voices/issues/1121
+    asset_id        blob,     -- asset policy hash (id) (28 byte binary hash)
+    asset_name      text,     -- name of the asset policy (UTF8) (32 bytes)
```

Comment:
```suggestion
    asset_name      blob,     -- name of the asset policy (UTF8) (32 bytes)
```
`asset_name` is correct.  As this is one named asset within a policy.
There can be multiple named assets under a single policy.
However, the `asset_name` must be stored as a blob, as it is just a variable length byte string on-chain.

You need to correct the type here, and make the type `Vec<u8>` throughout the code where it is used.
It should only be converted to a string when a response containing it is serialized.